public/app/themes/justice/inc/admin.php

```diff
@@ -36,4 +39,23 @@ public static function removeCustomizer()
         $customizer_url = add_query_arg('return', urlencode(remove_query_arg(wp_removable_query_args(), wp_unslash($_SERVER['REQUEST_URI']))), 'customize.php');
         remove_submenu_page('themes.php', $customizer_url);
     }
+    
+    /**
+     * Increase the dropdown limit on the block editor parent page field
+     *
+     * @param array $args An array of arguments for WP_Query
+     * @param WP_REST_Request $request The REST API request
+     *
+     * @return array $args The modified WP_Query array
+     */
+    public static function increaseDropdownLimit($args, $request)
+    {
+        if (is_user_logged_in()) {
+            if ($request->get_query_params()['context'] === 'edit') {
```

Comment:
Super minor one these could be on the same line.

Reply:
Standards would suggest different lines @EarthlingDavey, for readability purposes.
We follow PSR-12 to coding standards, sorry if I'd not ever mentioned that 😵‍💫 
https://www.php-fig.org/psr/psr-12/

Reply:
@wilson1000 I was thinking change it to one if statement with `&&` all on one line. 

Admittedly, I'm 100% not following psr-12 but I think that's allowed.

> Expressions in parentheses MAY be split across multiple lines